test/manipulation.test.js

```diff
@@ -1860,8 +1863,7 @@ describe('manipulation', function() {
       });
     });
 
-    bdd.itIf(connectorCapabilities.supportStrictDelete !== false, 'should allow delete(id) - ' +
-    'fail with error', function(done) {
+    it('should allow delete(id) - ' + 'fail with error', function(done) {
```

Comment:
`supportStringDelete` flag shouldn't be removed along with the `cloudantCompatibility` flag.